src/blockchain_txn_add_validator_stake_v1.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+package helium;
+
+message blockchain_txn_add_validator_stake_v1 {
```

Comment:
This entire txn type is unused in blockchain-core afaict. Also seems to be missing `owner_signature` field?

Reply:
Would this be used to *increase* validator stake?